tools/configure/configure.ml

```diff
@@ -133,7 +134,7 @@ let check_findlib_version prefs { CamlConf.findlib_version } =
     70: ".ml file without .mli file" bogus warning when used generally
 *)
 
-let coq_warnings = "-w +a-4-9-27-41-42-44-45-48-58-67-68-70"
+let coq_warnings = "-w -a+1..3-4+5..8-9+10..26-27+28..40-41-42+43-44-45+46..47-48+49..57-58+59..66-67-68+69-70"
```

Comment:
I don't get why this line contains negative numbers. It should be `-a` followed by positives.

Reply:
`-a` is needed to guard against future warnings, the rest is because I am not able to  parse the list properly otherwise, due to lack of readability for me.

Reply:
Can we have a comment explaining that the `-27` doesn't doe anything and that it's only there for readability?

Reply:
All the `-n` are doing noting (not just 27), this is why I asked.

Reply:
Of course, I just picked one.